ddtrace/contrib/django/middleware.py

```diff
@@ -18,6 +18,27 @@
 log = logging.getLogger(__name__)
 
 
+class TraceExceptionMiddleware(object):
```

Comment:
it must inherit from `MiddlewareClass` available in this module (like `TraceMiddleware` is doing).